src/Util/ColumnParser.php

```diff
@@ -191,7 +190,6 @@ public function getType($field, $type)
         $collection = new Collection($reflector->getConstants());
 
         $validTypes = $collection->filter(function ($value, $constant) {
-            $value;
```

Comment:
I also don't like code be added just for linters to be happy.
If anything it should be commented why it is done.